src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -46,7 +47,7 @@
     private static final String REDHAT_RELEASE_FILE = "/etc/redhat-release";
     private static final String[] DEFAULT_REDHAT_VARIANTS = {"rhel", "fedora"};
 
-    protected void detect(Properties props) {
+    protected void detect(Properties props, List<String> classifierWithLikes) {
```

Comment:
I actually want it to be a list since I want it clear that iteration order matters.